src/components/Home/BestSelling.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./css/bestselling.css";
+import { Link } from "react-router-dom";
+const BestSelling = (props) => {
+  return (
+    <>
+      <section>
+        <div className="best-sell-cont">
+          <img src={props.imgSrc} alt="" />
```

Comment:
Give value to `alt` attribute of image or else react would give a warning for the same